Sources/StringsLintFramework/Parsers/StructuredPlaceholderCommentParser.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mark Hall on 2022-07-19.
+//
+
+import Foundation
+
+public struct StructuredPlaceholderCommentParser: LocalizableParser {
+
+    public static var identifier: String {
+        return "structured_placeholder_comment_parser"
+    }
+
+    public var supportedFileExtentions: [String] {
```

Comment:
we also need to support structured comments in stringdict

Reply:
I think that will require a new parser. I will look into that once I'm done with the regular strings parser.

Reply:
See the changes to `StringsdictParser` to parse comments on `StringsDict` strings. We can re-use the same rule and rule configuration to do the actual linting.